sdk/keyvault/keyvault-admin/README.md

```diff
@@ -1,30 +1,90 @@
 
-[Source code](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/keyvault/keyvault-admin) | [Package (npm)](https://www.npmjs.com/package/@azure/keyvault-admin) | [API Reference Documentation](https://docs.microsoft.com/javascript/api/@azure/keyvault-admin) | [Product documentation](https://azure.microsoft.com/en-us/services/key-vault/) | [Samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/keyvault/keyvault-admin/samples)
+[Source code][package-gh] | [Package (npm)][package-npm] | [API Reference Documentation][docs] | [Product documentation][docs-service] | [Samples][samples]
 
 ## Getting started
 
-**Prerequisites**: You must have an [Azure subscription](https://azure.microsoft.com/free/) and a
-[Key Vault resource](https://docs.microsoft.com/en-us/azure/key-vault/quick-create-portal) to use this package.
-
-If you are using this package in a Node.js application, then use Node.js 8.x or higher.
-
 ### Install the package
 
-Install the Azure Key Vault Admin client library using npm:
-
-`npm install @azure/keyvault-admin`
+Install the Azure Key Vault administration client library for JavaScript and TypeScript with [NPM][NPM]:
 
-### Install the identity library
-
-Key Vault clients authenticate using the Azure Identity Library. Install it as well using npm
+```PowerShell
+npm install @azure/keyvault-admin
+```
 
-`npm install @azure/identity`
+### Prerequisites
+
+To work with the Azure Key Vault Administration client, the following is necessary:
+
+* An [Azure subscription][azure-sub].
+* An existing Azure Key Vault. If you need to create an Azure Key Vault, you can use the [Azure CLI][azure-cli].
+* If you are using this package in a Node.js application, then use Node.js 8.x or higher.
+
+#### Getting Azure credentials
+
+Use the [Azure CLI][azure-cli] snippet below to create/get client secret credentials.
+
+ * Create a service principal and configure its access to Azure resources:
+    ```PowerShell
+    az ad sp create-for-rbac -n <your-application-name> --skip-assignment
+    ```
+    Output:
+    ```json
+    {
+        "appId": "generated-app-ID",
+        "displayName": "some-app-name",
+        "name": "http://some-app-name",
+        "password": "random-password",
+        "tenant": "tenant-ID"
+    }
+    ```
+* Take note of the service principal objectId
+    ```PowerShell
+    az ad sp show --id <appId> --query objectId
+    ```
+    Output:
+    ```
+    "<your-service-principal-object-id>"
+    ```
+* Use the returned credentials above to set  **AZURE_CLIENT_ID** (appId), **AZURE_CLIENT_SECRET** (password), and **AZURE_TENANT_ID** (tenant) 
+environment variables. For example, you can use the [dotenv][dotenv] project and add a `.env` file to your project with hte following contents (for the full section of environment variables, see [Environment Variables](#environment-variables)):
```

Comment:
Based on the discussion in https://github.com/Azure/azure-sdk-for-js/pull/11075#discussion_r485386025, I would refrain from talking about dotenv and sample.env files.